conference/react-js/src/components/Talks/TalksTable/TalksTable.jsx

```diff
@@ -91,23 +91,19 @@ const getRows = (talks, columns) => (
 const TalksTable = ({ listOfTalks, columns, onClick, sortTalks }) => (
   <div className="table-container">
     <Table
-      selectable
-      multiSelectable
       onCellClick={onClick}
     >
       <TableHeader
-        displaySelectAll
-        adjustForCheckbox
-        enableSelectAll
+        displaySelectAll={false}
```

Comment:
You can remove this both attributes. Result must be the same

Reply:
Checkbox at header appear when I remove it